src/main/frontend/idb.cljs

```diff
@@ -13,12 +13,12 @@
 ;; To maintain backward compatibility
 
 
-(defonce store (Store. "localforage" "keyvaluepairs" 2))
```

Comment:
I had to move the initialization of this from require time to start time as explained in https://github.com/logseq/logseq/pull/6911/commits/441a19fcb891dc69551150ab623b1db41f4f3750 as it fails async unit tests due to no indexedDB

@tiensonqin Any specific things I should QA for the browser app for this change? I used the browser app and was able to write to a file fine